src/bin/driver.rs

```diff
@@ -179,10 +162,11 @@ mod app {
     #[shared]
     struct Shared {
         network: NetworkUsers<Settings, Telemetry>,
-
         settings: Settings,
         telemetry: TelemetryBuffer,
         signal_generator: [SignalGenerator; 2],
+        ltc2320: driver::ltc2320::Ltc2320,
```

Comment:
Please review the names in light of the existing ones.
remote_adc? Or header_adc?